tests/vmi_networking_test.go

```diff
@@ -722,42 +722,46 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			Expect(err).ToNot(HaveOccurred())
 			Expect(len(serverVMI.Status.Interfaces)).To(Equal(1))
 
-			serverIP := serverVMI.Status.Interfaces[0].IP
-			if netutils.IsIPv6String(serverIP) {
-				By("Checking traceroute from vmi to cluster nodes gateway")
-				// Cluster nodes subnet (docker network gateway)
-				// Docker network subnet cidr definition https://bit.ly/2wZTgMK
-				err := tests.CheckForTextExpecter(serverVMI, createExpectTraceroute6("2001:db8:1::1"), 30)
-				Expect(err).ToNot(HaveOccurred(), "Failed to traceroute to VMI %s within the given timeout", serverVMI.Name)
-			} else {
-				if ipv4NetworkCIDR == "" {
-					ipv4NetworkCIDR = api.DefaultVMCIDR
+			for _, serverIP := range serverVMI.Status.Interfaces[0].IPs {
+				if netutils.IsIPv6String(serverIP) {
+					Skip("Skip till https://github.com/kubevirt/kubevirtci/pull/432, https://github.com/kubevirt/kubevirt/pull/3957 are merged")
```

Comment:
I would remove https://github.com/kubevirt/kubevirt/pull/3957 from the skip message. All this PR is depends on it. So I would update the PR message and add a hold to the PR until the mentined PR is merged.

Reply:
hold you mean here or on #3957 ?
because i think i can remove 3957 mentioned here, but we dont need any hold?
once all the other needed PRs are merged, the skip will be safe the remove and whoever need this code, will have it already while developing, wdyt?

beside that the method is fine that we will use it on all the other tests that have the same phenomena ?

Reply:
updated title, left just 432

Reply:
i dont think we should wait, see please https://github.com/kubevirt/kubevirt/pull/3989#discussion_r472213579